requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Setup
 libyaz0>=0.5
 colorama>=0.4.3
+pyelftools>=0.26
```

Comment:
Out of curiosity what is this for? Is it important enough that it should get its own PR so it gets in sooner?

Reply:
Oh right, I totally forgot about this. `pyelftools` is used by `z64compress_wrapper.py`, so yeah we should definitely move it to its own PR.